client/src/templates/Challenges/redux/execute-challenge-saga.js

```diff
@@ -197,8 +197,8 @@ function* buildChallengeData(challengeData, options) {
 function* executeTests(testRunner, tests, testTimeout = 5000) {
   const testResults = [];
   for (let i = 0; i < tests.length; i++) {
-    const { text, testString } = tests[i];
-    const newTest = { text, testString };
+    const { text, testString, running } = tests[i];
+    const newTest = { text, testString, running };
```

Comment:
```suggestion
    const { text, testString } = tests[i];
    const newTest = { text, testString, running: false };
```
At this point we know that whatever happens, the next update to the tests will be that they've stopped running.